status/kube.sh

```diff
@@ -0,0 +1,19 @@
+# Requires https://github.com/jonmosco/kube-tmux
+
+show_kube() {
+  local index icon color text context_color namespace_color escaped_context_color escaped_namespace_color symbol_enabled module
+
+  index=$1
+  icon=$(get_tmux_option "@catppuccin_kube_icon" "󱃾")
+  color=$(get_tmux_option "@catppuccin_kube_color" "$thm_blue")
+  context_color=$(get_tmux_option "@catppuccin_kube_context_color" "$thm_red")
+  namespace_color=$(get_tmux_option "@catppuccin_kube_namespace_color" "$thm_cyan")
+  escaped_context_color=${context_color//#/\\#}
+  escaped_namespace_color=${namespace_color//#/\\#}
```

Comment:
Does it really have to be escaped? Don't really see the reason why.
Sorry for the question don't have a kubernetes cluster to test.